modules/migration-manual-rollback.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+[id='migration-manual-rollback_{context}']
+= Manual rollback
+
+You can manually roll back a failed migration. Manual rollback is only required if an application has been quiesced, to prevent data corruption in the PV.
+
+This procedure is not required if the application was not quiesced, because the original application is still running on the source cluster.
+
+.Procedure
+
+. On the target cluster, switch to the migrated project:
++
+----
+$ oc project <project>
+----
+
+. Get the deployed resources:
++
+----
+$ oc get all
+----
+
+. Delete each deployed resource type or scale it to `0`:
++
+----
+$ oc delete <resource_type>
+$ oc scale --replicas=0 <resource_type>/<resource_name>
+----
++
+You must ensure that no applications are running on the target cluster and accessing data on the PVC.
+
+. To stop a DaemonSet without deleting it, update the `nodeSelector` in the YAML file:
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: hello-daemonset
+spec:
+  selector:
+      matchLabels:
+        name: hello-daemonset
+  template:
+    metadata:
+      labels:
+        name: hello-daemonset
+    spec:
+      nodeSelector:
+        role: worker <1>
+----
+<1> Specify a `nodeSelector` value that does not exist on any node.
+
+. During migration, the reclaim policy for bound PVs is `Retain`, to ensure that data is not lost when an application is removed from the source cluster. You can update the PV's reclaim policy:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv0001
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain <1>
+  ...
+status:
+  ...
+----
+<1> Specify `Recycle` or `Delete`.
+
+. On the source cluster, switch to the migrated project and get its deployed resources:
++
+----
+$ oc project <project>
+$ oc get all
+----
+
+. Start one or more replicas of the deployed resources:
```

Comment:
Do you need to restart each deployed resource? If so, please s/of the deployed resources/of each deployed resource

Reply:
I would assume so. Will do.